src/pages/login/index.vue

```diff
@@ -88,6 +93,31 @@ async function Login() : Promise<void> {
   }
 };
 
+/**
+ * Google login callback
+ */
+
+ interface GoogleResponseObject {
+  clientId: string;
+  client_id: string;
+  credential: string;
+  select_by: string;
+}
+
+const callback = async (response : GoogleResponseObject) => {
+  let credential = response.credential;
+
+  const { data: UserCreated } = await useApi(UsersApi, 'usersUserGoogleOauth2Create')(() => ({
+      googleSocialAuth: {
+        'auth_token': credential,
+      }
+  }));
```

Comment:
Aquí habría que pasarle como opción al composable un { skipCache: { request: true }} porque es una solicitud atómica que no se beneficia del estado y no necesita cachearse.

```suggestion
  const { data: UserCreated } = await useApi(UsersApi, 'usersUserGoogleOauth2Create')(() => ({
      googleSocialAuth: {
        'auth_token': credential,
      }
  }));
```